js/binance.js

```diff
@@ -835,16 +939,23 @@ module.exports = class binance extends Exchange {
         }
         await this.loadMarkets ();
         const market = this.market (symbol);
-        const origClientOrderId = this.safeValue (params, 'origClientOrderId');
+        const marketType = this.options['defaultMarket'];
+        let method = 'GetOrder';
+        if (marketType === 'futures') {
+            method = 'fapiPrivate' + method;
```

Comment:
we can avoid the string concatenation here and replace it with:

```
let method;
if (marketType === 'futures') {
    method = 'fapiPrivateGetOrder';

Reply:
Yep, that would be great! Constant literals would work better and faster )

Reply:
> > How to use binance future api by ccxt?
> 
> You will need to set the `defaultMarket` option to `futures`.
> 
> ```js
> let binance_futures = new ccxt.binance({ options: { defaultMarket: 'futures' } })
> ```

How to use this with Python?

Reply:
@szdong 

- build it as described here: 
- then: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-build--run-tests-on-your-local-machine

```Python
binance_futures = ccxt.binance{{"options":{"defaultMarket":"futures"}})
# ...
```